packages/fx-core/src/core/error.ts

```diff
@@ -265,7 +265,8 @@ export function ProjectSettingError(): UserError {
 
 export function UpgradeCanceledError(): UserError {
   return new UserError(
-    "UpgradeCanceledError",
+    // @see tools.isUserCancelError()
+    "User Cancel",
```

Comment:
Use "UserCancel" instead of "User Cancel"? Because other error codes use CamelCase style and `tools.isUserCancelError()` also supports it.

Reply:
ok